<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cross</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxOption">
     <property name="title">
      <string>Option</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBoxIn">
        <item>
         <property name="text">
          <string>Input</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VCF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PED/MAP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HapMap</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>General</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditIn"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonIn">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBoxOut">
        <item>
         <property name="text">
          <string>Output</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VCF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PED/MAP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HapMap</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>General</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditOut"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonOut">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxSort">
        <property name="text">
         <string>Sort position</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEditLog">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditIn</tabstop>
  <tabstop>pushButtonIn</tabstop>
  <tabstop>lineEditOut</tabstop>
  <tabstop>pushButtonOut</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
